src/components/__internal__/abstract-custom-field/abstract-custom-field.css

```diff
@@ -2,16 +2,16 @@
 @import '../../../styles/colors-v2.css';
 @import '../../../styles/spacing.css';
 @import '../../../styles/typography.css';
+@import '../../../styles/typography-v2.css';
 
 .input-styles {
   border: 1px solid var(--mds-grey-12);
   border-radius: var(--spacing-x-small);
   box-sizing: border-box;
   color: var(--mds-grey-87);
-  font-size: var(--mavenlink-type-base-size); /* stylelint-disable-line mds/typography */
-  padding-top: 7px;
+  font: var(--mds-type-content);
+  height: 32px;
```

Comment:
Should we codify this as a variable to make it less of a magic number? 🤔

Reply:
Oh, yeah. That is x-large already. Hah.